src/quo2/components/selectors/filter/view.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.selectors.filter.view
+  (:require [quo2.components.icon :as icon]
+            [quo2.components.selectors.filter.style :as style]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn view
+  [initial-props]
+  (let [pressed? (reagent/atom (:pressed? initial-props))]
+    (fn [{:keys [blur? override-theme on-press-out]
+          :or   {override-theme (theme/get-theme)}}]
+      [rn/touchable-without-feedback
+       {:test-ID             :selector-filter
```

Comment:
for component tests we idea use accessibility label.
`getByLabelText` method should work 👍

Reply:
Cool, it works indeed!